src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx

```diff
@@ -29,28 +29,25 @@ export const ScheduleItem: React.FunctionComponent<PropsType> = ({
   return (
     <div className="schedule-item-container">
       <div className={`time-section ${isCurrentEvent ? "primary" : ""}`}>
-        <div>
-          <b>{formatUtcSeconds(event.start_utc_seconds)}</b>
-        </div>
-        <div>
+        <span className="event-date">
+          {labelDayRoomSchedule(event.start_utc_seconds)}
+        </span>
+        <span className="event-time">
+          {formatUtcSeconds(event.start_utc_seconds)}
+        </span>
+        <span className="event-time">
```

Comment:
we prefer BEM-styled class names, so could we have `schedule-item-container__event-date`, `schedule-item-container__event-time`.

And also would be just awesome, if you could rename `schedule-item-container` -> `ScheduleItem` (and change other class names accordingly: `ScheduleItem__event-date`, `ScheduleItem__event-time`, etc). Our approach is to give the base class the same name as its component has.